sat/config.go

```diff
@@ -136,6 +140,11 @@ func ConfigFromRunnableArg(runnableArg string) (*Config, error) {
 		port = fmt.Sprintf("%d", randPort.Int64()+1000)
 	}
 
+	procUUID, ok := os.LookupEnv("SAT_UUID")
```

Comment:
should we have a validation here? If SAT_UUID is declared, but the value is not an uuid value, then this should exit or generate a new uuid for itself, not sure yet

```golang
parsedUUID, err := uuid.Parse(procUUID)
if err != nil {
    return nil, errors.Wrap(err, "uuid was in environment, but wasn't correct format")
}
```

Reply:
Good call, added the check.